app/views/conference/schedule.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <% if @conference.program.events.scheduled.any? %>
+  <% if @rooms.length > 1 and @conference.program.events.scheduled.any? %>
```

Comment:
@nishanthvijayan yes, if there is only one room the all events approach is better, as it lists all the events by date and time and in that case the room is not important.